.github/workflows/integration-tests-on-release.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
           tests/scripts/github-action-helper.sh create_helm_tag
           tests/scripts/helm.sh up
           export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
-          SKIP_TEST_CLEANUP=false SKIP_CLEANUP_POLICY=false go test -v -timeout 1800s -run CephHelmSuite github.com/rook/rook/tests/integration
+          SKIP_TEST_CLEANUP=false SKIP_CLEANUP_POLICY=false go test -v -timeout 2400s -run CephHelmSuite github.com/rook/rook/tests/integration
```

Comment:
For now let's just increase the timeout on the object store suite, instead of all the suites in this file. Later, we can increase the timeouts on other suites if needed.

Reply:
Make sense